src/main/java/pl/sdacademy/pewex/product/controller/ProductController.java

```diff
@@ -1,4 +1,33 @@
 package pl.sdacademy.pewex.product.controller;
 
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import pl.sdacademy.pewex.product.model.ProductDetail;
+import pl.sdacademy.pewex.product.model.ProductListDTO;
+import pl.sdacademy.pewex.product.service.ProductService;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/product")
 public class ProductController {
+
+    private final ProductService productService;
+
+    public ProductController(ProductService productService) {
+        this.productService = productService;
+    }
+
+    @GetMapping("/list")
```

Comment:
Na pewno list jest potrzebne?

Reply:
Wiem, że nie jest potrzebne. W moim prywatnym odczuciu tak jest czytelniej, ale jeśli to nie jest dobra praktyka, można to zmienić.